patroni/postgresql.py

```diff
@@ -154,7 +155,8 @@ def get_server_parameters(self, config):
         parameters = config['parameters'].copy()
         listen_addresses, port = (config['listen'] + ':5432').split(':')[:2]
         parameters.update({'cluster_name': self.scope, 'listen_addresses': listen_addresses, 'port': port})
-        return parameters
+        major_version = self._major_version or 9.1
```

Comment:
Should be 9.2, as our code doesn't work with 9.1 (it lacks support for row_to_json and array_to_json at least).